### PR TITLE
(docs) Outdated links (2) on README.md

### PR DESCRIPTION
**Pull request by ehom**
Filed Ticket indicating 2 outdated links.
Updated the affected links.

Also:
* beautified some of the sentences, adding commas for improved readability.
* shortened a few instances of hyperlinked text.
* fixed casing on a few phrases.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,24 +25,24 @@ Installation
...
+The PE documentation is available [here](https://puppetlabs.com/docs/pe/latest).
 
 To install an open source release of Puppet,
-[see the installation guide on the docs site.](https://docs.puppetlabs.com/puppet/latest/reference/install_pre.html)
+see the [Installation Guide](https://docs.puppetlabs.com/puppet/latest/reference/install_pre.html) on the docs site.
 
-If you need to run Puppet from source as a tester or developer,
-[see the running from source guide on the docs site.](https://docs.puppetlabs.com/guides/from_source.html)
+If you need to run Puppet from source as a tester or developer, see the
+[Running Puppet From Source](https://docs.puppetlabs.com/guides/from_source.html) Guide on the docs site.
```
Comment (jtappa) [inline README.md]: This page actually doesn't exist in modern docs, so this line can be removed. We shouldn't be linking to the 3.8 documentation any longer.

File: README.md [hunk 2]
```diff
@@ -25,24 +25,24 @@ Installation
 
 The best way to run Puppet is with [Puppet Enterprise](https://puppetlabs.com/puppet/puppet-enterprise),
 which also includes orchestration features, a web console, and professional support.
-[The PE documentation is available here.](https://docs.puppetlabs.com/pe/latest)
+The PE documentation is available [here](https://puppetlabs.com/docs/pe/latest).
 
 To install an open source release of Puppet,
-[see the installation guide on the docs site.](https://docs.puppetlabs.com/puppet/latest/reference/install_pre.html)
+see the [Installation Guide](https://docs.puppetlabs.com/puppet/latest/reference/install_pre.html) on the docs site.
```
Comment (jtappa) [inline README.md]: This link got missed, should be updated to `https://puppet.com/docs/puppet/latest/install_pre.html`

File: README.md [hunk 3]
```diff
@@ -25,24 +25,24 @@ Installation
 
 The best way to run Puppet is with [Puppet Enterprise](https://puppetlabs.com/puppet/puppet-enterprise),
 which also includes orchestration features, a web console, and professional support.
-[The PE documentation is available here.](https://docs.puppetlabs.com/pe/latest)
+The PE documentation is available [here](https://puppetlabs.com/docs/pe/latest).
```
Comment (jtappa) [inline README.md]: Should just be `https://puppet.com/docs/pe/latest` (without the labs)

**Comment by jtappa** [comment]
All of the changes look good, just needs a rebase so that all of the commits are squashed into one, and the commit summary also starts with `(docs)`

**Comment by ehom** [comment]
@jtappa thanks !!!

**Pull request merged**